### ISSUE TITLE
Remove libssh2 bbappend file

### ISSUE THREAD

**Issue by derekstraka** [meta-rust/meta-rust#61]
The file can be removed from master once [72946ac16b52a43096fc109fbbcf14c62dc11d9f](https://github.com/openembedded/meta-openembedded/commit/72946ac16b52a43096fc109fbbcf14c62dc11d9f) is merged into master.  Currently it is on master-next

**Comment by cardoe** [comment]
But it will need to remain for our krogoth branch no? So we should probably go ahead and branch krogoth now?

**Comment by derekstraka** [comment]
Yeah.  It will need to be in the krogoth branch.  Same goes for libgit2

**Comment by derekstraka** [comment]
This is now in master and I will create a PR once #59 is complete

**Comment by cardoe** [comment]
krogoth has been created
